west.yml

```diff
@@ -147,11 +147,6 @@ manifest:
       remote: nordicsemi
       revision: 24f1b2b0c64c694b7f9ac1b7eab60b39236ca0bf
       path: modules/lib/cddl-gen
-    - name: homekit
```

Comment:
I think you should also remove line 44 the group filter.

Reply:
Done:
https://github.com/nrfconnect/sdk-nrf/compare/71207bc3bec9089a1224346b40ef14161897c6ca..cd7c7c2b2ce30e3d2fdea4e7716afd2f076d4983